app/cafe.py

```diff
@@ -0,0 +1,23 @@
+import datetime
+from app.errors import (NotVaccinatedError, OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
```

Comment:
The function `visit_cafe` has a parameter `visitor` of type `dict`. It is a good practice to use a data class or a named tuple instead of a plain `dict`. It would make your code safer and more readable. You can define a `Visitor` class with the fields `vaccine`, `wearing_a_mask`, etc., and then use it as a type hint for the `visitor` parameter. This way, you will know exactly what fields the `visitor` has and what their types are.